tests/app/helpers.py

```diff
@@ -40,6 +40,9 @@ def setup_method(self, method):
         data_api_client.find_frameworks = mock.Mock()
         data_api_client.find_frameworks.return_value = self._get_frameworks_list_fixture_data()
 
+        self.app_env_var_mock = mock.patch.dict('gds_metrics.os.environ', {'PROMETHEUS_METRICS_PATH': '/_metrics'})
```

Comment:
Should the metrics path be `/admin/_metrics`?

```suggestion
        self.app_env_var_mock = mock.patch.dict('gds_metrics.os.environ', {'PROMETHEUS_METRICS_PATH': '/admin/_metrics'})
```

Reply:
See:
https://github.com/alphagov/digitalmarketplace-user-frontend/pull/119#discussion_r284611233

Reply:
Okay